week2/project/ecommerce/src/pages/ProductsPage/products/products.jsx

```diff
@@ -0,0 +1,46 @@
+import Product from './product'
+import React, { useState } from 'react'
```

Comment:
you can import `useEffect` here as well and use it directly in line 31
📝advise📝